pom.xml

```diff
@@ -23,7 +23,7 @@
     <tychoExtrasVersion>${tychoVersion}</tychoExtrasVersion>
     
     <!-- Fuse version -->
-    <fuse-tooling-url>http://download.jboss.org/jbosstools/mars/development/updates/integration-stack/fuse-tooling/8.0.0.Beta2/all/repo/</fuse-tooling-url>
```

Comment:
@pleacu So the growing consensus with @lhein and @apupier is to split out the "fuse-tooling-url" change into a separate JIRA-driven PR. Can we have you update this PR to remove that change and put it into a separate PR with a new JIRA?